evm-contracts/src/v0.6/tests/VRFCoordinatorMock.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.6.6;
```

Comment:
This looks sane, but we probably want an integration test at some point (not in this PR) like [TestD20ExampleContract](https://github.com/smartcontractkit/chainlink/blob/feature%2F%23175865202-d20-example/core/services/vrf/vrf_testnet_d20_test.go).